services/validate.js

```diff
@@ -19,12 +17,19 @@ fs.readdirSync(baseDir)
   })
   .forEach(function (fileName) {
     try {
-      validator.addSchema(fileName, require(path.join(baseDir, fileName)))
+      validator.addSchema(JSON.parse(path.join(baseDir, fileName)), fileName)
```

Comment:
I've opened a PR for the [quote issue](https://github.com/interledgerjs/ilp-connector/pull/263), and other for an issue in the [notary](https://github.com/interledgerjs/five-bells-notary/pull/84).

I also ran into another bug:

```diff --git a/lib/validator.js b/lib/validator.js
index 426efc4..358a9f7 100644
--- a/lib/validator.js
+++ b/lib/validator.js
@@ -97,7 +97,7 @@ class Validator {
    * @returns {void}
    */
   validateUriParameter (paramId, paramValue, schema) {
-    const isValid = this.ajv.validate(paramValue, schema + '.json')
+    const isValid = this.ajv.validate(schema + '.json', paramValue)
     if (!isValid) {
       throw new InvalidUriParameterError(paramId + ' is not a valid ' + schema,
         this.ajv.errors)
@@ -120,7 +120,7 @@ class Validator {
     const json = yield parse(ctx)

     if (schema) {
-      const isValid = this.ajv.validate(json, schema + '.json')
+      const isValid = this.ajv.validate(schema + '.json', json)
       if (!isValid) {
         throw new InvalidBodyError('JSON request body is not a valid ' + schema,
           this.ajv.errors)
```